data/anshsaxena1703.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "Ansh Saxena",
+  "bio": "Open Source Developer | Full Stack Developer | Video Editor"
+  "tags" : [
+    "Open Source",
+    "Web Development",
+    "Cloud Computing",
+    "Machine Learning",
+  ]
```

```suggestion
  ],
```
Please add comma here